src/harness/certs/generate_fake_certs.sh

```diff
@@ -363,7 +360,7 @@ openssl ca -cert non_cbrs_root_ca.cert -keyfile private/non_cbrs_root_ca.key -in
 echo "\n\nGenerate sas certificate/key"
 openssl req -new -newkey rsa:2048 -nodes \
     -reqexts sas_client_mode_req -config ../../../cert/openssl.cnf \
-    -out non_cbrs_signed_sas.csr -keyout non_cbrs_signed_sas.key \
+    -out non_cbrs_signed_sas.csr -keyout private/non_cbrs_signed_sas.key \
```

Comment:
Why moving it to private? it breaks test SSS_9

Reply:
I think this is not a functional modification, but all the private keys are generated in private/ folder, so I made this change to be coherent with the rest of the code, don't you agree?

Reply:
No, only private keys that are not used directly by the fake_sas.py or by testcases are put in the private directory (like the intermediate CA keys need to generate leaf certificates). 
Note also that at the end of the generate_fake_certs.sh script we delete this 'private' directory.

The non_cbrs_signed_sas.key private key is used directly by test SSS_9 so it have to be keep in the src/harness/certs directory

Reply:
Ah, I thought it was not used, sorry!  then I return it back to the folder.